<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGroup</class>
 <widget class="QDialog" name="DialogGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить группу</string>
  </property>
  <widget class="QLabel" name="lab_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Группа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_group">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_group_year">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_group_year">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_group">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
